sig/line/bot/v2/http_client.rbs

```diff
@@ -14,85 +14,85 @@ module Line
 
         def get: (
             path: String,
-            ?query_params: Hash[String, untyped]?,
-            ?headers: Hash[String | Symbol, untyped]?
+            ?query_params: Hash[Symbol, untyped]?,
+            ?headers: Hash[Symbol, untyped]?
           ) -> Net::HTTPResponse
 
         def post: (
             path: String,
-            ?query_params: Hash[String, untyped]?,
-            ?body_params: (File | Hash[untyped, untyped])?,
-            ?headers: Hash[String | Symbol, untyped]?
+            ?query_params: Hash[Symbol, untyped]?,
+            ?body_params: (File | Hash[untyped, untyped] | untyped)?,
```

Comment:
- `untyped` is for classes we define (like `PushMessageRequest`)
- I want to keep `File` and `Hash` for hint, though we can delete this